server/src/features/FoldingRangesRequestHandler.ts

```diff
@@ -0,0 +1,40 @@
+import { FoldingRange, FoldingRangeParams } from 'vscode-languageserver';
+
+import { YmlDefinitionProvider } from '../definitions';
+import { AbstractYmlObject } from '../yml-objects';
+
+/**
+ * Create a request handler for the event `foldingRanges`.
```

Comment:
Is it better like this @mminot-yseop?

Reply:
https://github.com/yseop/vscode-yseopml/pull/177#commitcomment-44732210 xd